scheduler/src/cook/kubernetes/api.clj

```diff
@@ -716,8 +716,11 @@
                                               (contains? checkpoint-failure-reasons (:reason/name reason)))
                                             instance)]
             (if (-> checkpoint-failures count (>= max-checkpoint-attempts))
-              (log/info "Will not checkpoint task-id" task-id ", there are at least" max-checkpoint-attempts "failed instances"
-                        {:job job})
+              (log/info "Will not checkpoint, there are too many checkpoint failures"
```

Comment:
```suggestion
              (log/info "Will not checkpoint, there are at least" max-checkpoint-attempts "checkpoint failures"
```

Reply:
Why not just have `max-checkpoint-attempts` in the dictionary?

Reply:
yeah, it's already in the dictionary. I just want this wording is to make it clear that it's greater than or equal to.

Reply:
Done